packages/agent/src/polling/strategy.ts

```diff
@@ -135,3 +135,33 @@ export function chain(...strategies: PollStrategy[]): PollStrategy {
     }
   };
 }
+
+
+export type Delay = number;
+export type BackoffStrategy = (retryCount: number) => Delay;
+
+/**
+ * An exponential backoff strategy.
+ * @param iterations The number of retries that have been done.
+ */
+export function exponentialBackoff(iterations: number): Delay {
+  if (iterations === 0) {
+    return 0;
+  }
+  return 2 ** iterations * 150;
```

Comment:
I recommend introducing jitter here.

https://aws.amazon.com/blogs/architecture/exponential-backoff-and-jitter/

Exponential backoff policy that we use in dfx and the rust agent: https://github.com/ihrwein/backoff/blob/master/src/exponential.rs#L111

Reply:
Jitter is interesting - it'll make the tests a little more complicated, but I see the utility of it